modules/dop/services/sceneset/sceneset.go

```diff
@@ -67,6 +63,24 @@ func (svc *Service) CreateSceneSet(req apistructs.SceneSetRequest) (uint64, erro
 	return sceneSet.ID, nil
 }
 
+func (svc *Service) sceneSetNameCheck(spaceID uint64, name string, setID uint64) bool {
```

Comment:
you'd better to declare the items to check.
like `sceneSetCaseSensitiveNameCheck`

Reply:
good idea